src/input/tablet.c

```diff
@@ -50,6 +73,10 @@ handle_axis(struct wl_listener *listener, void *data)
 
 		double x = tablet->x;
 		double y = tablet->y;
+		adjust_for_tablet_area(tablet->tablet->width_mm, tablet->tablet->height_mm,
+			rc.tablet.tablet_area.top_mm, rc.tablet.tablet_area.right_mm,
```

Comment:
If we use `wlr_fbox` we can pass one argument here rather the four (i.e. `top_mm`, `right_mm`, `bottom_mm`, `left_mm`). Makes it easier to read.